Backend/Backend.UnitTest/EnterpriseControllerTest.cs

```diff
@@ -62,14 +65,70 @@ public void GetEnterprises_ReturnsEnterprisesInAlphabeticalOrderByName()
         [TestMethod]
         public void GetEnterprise_ReturnsSingleEnterprise()
         {
-            //Arrange
-            _fakeContext.EnterpriseList.Add(new Enterprise { Id = 5, Name = "Enterprise1", Password = "abc", CoverImageURL = "myimage.com", ModifiedUTC = 2 });
+            //Arrange                           --Is there an easier way?
+            MediaItemType mediaItemType = new MediaItemType { Id = 25, Name = "newMediaItemType" };
+            MediaItem mediaItem = new MediaItem { Id = 20, Name = "newMediaItem", PlaceId = 15, MediaItemType = mediaItemType };
+            Place place = new Place { PlaceId = 15, Name = "newPlace", ParticipantId = 10, MediaItems = new List<MediaItem>() { mediaItem } };
+            Participant participant = new Participant { Id = 10, ParticipantName = "newParticipant", EnterpriseId = 5, Places = new List<Place>() { place } };
+            Enterprise enterprise = new Enterprise { Id = 5, Name = "Enterprise1", Password = "abc", CoverImageURL = "myimage.com", ModifiedUTC = 2, Participants = new List<Participant>() { participant } };
+
+            _fakeContext.EnterpriseList.Add(enterprise);
+
+            _target.Request = new HttpRequestMessage();
+            _target.Request.Headers.Add("Authorization", "abc");
 
             //Act
-            var response = _target.Get(5, "abc");
+             var response = _target.Get(5);
             
             //Assert
-            response.Name.Should().Be("Enterprise1");
+            response.Participants.First()
+                    .Places.First()
+                    .MediaItems.First()
+                    .MediaItemType.Name.Should().Be("newMediaItemType");
+        }
+
+        [TestMethod]
+        public void GetEnterprise_ReturnError403()
+        {
+            //Arrange
+            Enterprise enterprise = new Enterprise { Id = 5, Name = "Enterprise1", Password = "abc", CoverImageURL = "myimage.com", ModifiedUTC = 2 };
+
+            _fakeContext.EnterpriseList.Add(enterprise);
+
+            _target.Request = new HttpRequestMessage();
+            _target.Request.Headers.Add("Authorization", "wrongPassword");
+
+            //Act
+            try
+            {
+                var response = _target.Get(5);
+            }
```

Comment:
When I do tests like this I usually put Assert.Fail() at the end of the try to make sure that an exception really is thrown!